docs/zh_cn/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```

Comment:
What's `make.bat` used for?

Reply:
It's for generating html on windows by running `./make.bat html`.

Reply:
Is it a must?

Reply:
not a must, but is for windows users to check readthedocs on local machine.